core/modules/system/system.module

```diff
@@ -3496,6 +3496,7 @@ function system_autoload_info() {
     'Archive_Tar' => 'system.tar.inc',
     'ModuleUpdater' => 'system.updater.inc',
     'ThemeUpdater' => 'system.updater.inc',
+    'BackdropSetMessageTest' => 'tests/system.test',
```

Comment:
@quicksketch I wasn't sure here; no other classes in the `tests/system.test` file seem to be mentioned in any `hook_autoload_info()` as far as I can tell.

So either this one shouldn't be either; or they all need to be added?